tests/README.md

```diff
@@ -0,0 +1,46 @@
+d# Setup
+
+Make sure you have a correct configured kubectl and all env variable are set.
+To set up another provider, have a look at the [official documentation](https://paritytech.github.io/zombienet/).
+
+To run the network:
+```
+zombienet spawn network-config.toml
+```
+
+There are two ways, to execute a test. If you do not have a spawned network, you can simply execute:
+
+```
+zombienet tests FILENAME.zndsl
+```
+
+This will create the network and perform the tests. After all tests are finished, the network is destroyed.
+
+If you already have a spawned network, you have to look up the ´runningNetworkSpec´ which is typically in  /tmp/zombie-{HASH}/zombie.json located.
+
+An example call would be:
+
+
+```
+zombienet test {FILENAME}.zndsl  /tmp/zombie-{HASH}/zombie.json
+```
+
+## Known Issues
+
+### Kubernetes provider
+
+1. If you face this error:
+
+```
+Error:          Error: Command failed with exit code 1: kubectl --kubeconfig /home/{USER}/.kube/config --namespace zombie-087286c76d4301bac2d39f96a4a97698 cp temp-collator:/cfg/genesis-state /tmp/zombie-087286c76d4301bac2d39f96a4a97698_-1512996-vZpBh73wdvtR/2000/genesis-state
+error: unable to upgrade connection: container not found ("temp-collator")
+Defaulted container "temp-collator" out of: temp-collator, transfer-files-container (init)
+```
+
+Make sure, you have export all env variable.
```

Comment:
What are these variables you are talking about? It is not very clear, at least to me.

Reply:
I gave more explanation in the env-example file